multicluster/config/certmanager/certificate.yaml

```diff
@@ -0,0 +1,25 @@
+# The following manifests contain a self-signed issuer CR and a certificate CR.
```

Comment:
Question - for the yamls in multicluster/config/, how I know which are generated by kubebuilder, which are not?

Reply:
I think all files under ./config are auto-generated, but modifiable.

Reply:
control-manager is key word that Kube-builder uses, we can change if we want.

muticluster-conrtroller/antrea:latest is changed by us, we want use any name. I did not follow the xxx/antrea-ubuntu convention because it is running in a distroless container.

Reply:
ms-pod name: antrea-multicluster-conrtroller
image name: antrea/antrea-multicluster-controller:latest

Reply:
done.